pkg/apis/kubeone/v1beta3/register.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The KubeOne Authors.
+Copyright 2021 The KubeOne Authors.
```

Comment:
We should change license headers to 2024 for v1beta3 (affects other files in this package too).

Reply:
Fixed